src/Microsoft.TemplateSearch.Common/Abstractions/TemplatePackageSearchData.cs

```diff
@@ -42,6 +44,12 @@ public TemplatePackageSearchData(ITemplatePackageInfo packInfo, IEnumerable<Temp
         /// <inheritdoc/>
         public long TotalDownloads { get; }
 
+        /// <inheritdoc/>
+        public IReadOnlyList<string> Owners { get; }
```

Comment:
Owners can change over time for a package. An existing owner and can add or remove owners. If I am understanding this correctly, there will be a cached data file built into the release distribution. If this data is stale, what is the impact on the scenario?

Reply:
Thanks, that a good point. 
Now this data is regenerated once every hour from scratch, so it will be updated. I'm working on incremental updates now, so I would ensure we update this property too.